tidb-cloud/serverless-export.md

```diff
@@ -0,0 +1,120 @@
+---
+title: TiDB Serverless Export (Beta)
```

Comment:
Since it's a how-to guide, how about starting with a verb in the title, like that of https://docs.pingcap.com/tidbcloud/backup-and-restore-serverless. A noun title sounds like a concept topic.
```suggestion
title: Export Data from TiDB Serverless (Beta)
```

Reply:
Currently, the export can also be done via mysqldump and Dumpling. Only exporting data using "TiDB Serverless Export" is in beta.😂